editor/store/effects.js

```diff
@@ -326,6 +326,11 @@ export default {
 		const { dispatch } = store;
 
 		let result;
+		// TODO: these are potentially undefined, this fix is in place
+		// until there is a filter to not use reusable blocks if undefined
+		if ( ! has( wp, 'api.models.Blocks' ) && ! has( wp, 'api.collections.Blocks' ) ) {
```

Comment:
Should this be `||` instead of `&&`?

Reply:
They are not exists, so both have to not exist in order to bail.

If ( ! A && ! B ) is the opposite of If ( A || B ) which would be the check to continue if one were defined.